mtags/src/main/scala/scala/meta/internal/mtags/ScalaToplevelMtags.scala

```diff
@@ -253,7 +253,6 @@ class ScalaToplevelMtags(
         withOwner(owner) {
           tpe(name.name, name.pos, Kind.CLASS, 0)
         }
-        term(name.name, name.pos, Kind.OBJECT, 0)
```

Comment:
I did the same thing in [`ScalaMtags`](https://github.com/scalameta/metals/blob/80b0dde6ec45659445a0359b2f76c66e835200b8/mtags/src/main/scala/scala/meta/internal/mtags/ScalaMtags.scala#L196).

Reply:
Fixed it there too!